pipeline/extract/get_sensor_readings.py

```
@@ -0,0 +1,71 @@
+import csv
+from requests import get
+from time import time
+import json
+
+
+def get_max_plant_id(base_url: str) -> int:
+    """Gets the ID from max plants on display from the API's base URL."""
+    response = get(base_url)
+    data = response.json()
+
+    if data and "plants_on_display" in data:
+        return data["plants_on_display"]
+    else:
+        print("Failed to fetch status information.")
+        return None
```

All functions should return something, or raise an exception. In this case you may want to return an exception, but you can only have one return type.